### DISCUSSION TITLE
【域名解析问题】无法使用 hosts 文件解析域名

### DISCUSSION THREAD

**Discussion by vzcrab** [SeaHOH/GotoX#191] [问答/咨询] [answered]
在 hosts 文件添加域名 ip地址关联，但日志中提示

```
L4:60735-> 无法解析主机：'****.****.****'，路径：'/'，请检查是否输入正确！
```

浏览器中返回的是 504 无法解析。

有什么方式能指定 hosts 文件为第一优先级，再使用 DNS 解析

**Comment by SeaHOH** [comment, marked as answer]
不支持系统 hosts 设置，但是可以在配置文件中设置。主配置的 iplist 和规则配置的直连规则。

**Reply by vzcrab** [reply]
感谢解答。

有添加这个功能的计划吗？使用 hosts 文件可以和其他一些改 hosts 的工具兼容。

**Reply by SeaHOH** [reply]
这个问题是改善系统查询性能时导致的，虽然我有打算在空闲时间重构整个程序，这个问题也会顺便得到解决，但我并不确定会在近期内完成。
